src/app/pages/FeedPage/components/TwitsUnit.tsx

```diff
@@ -0,0 +1,140 @@
+import 'index.css'
+
+import SvgLike from 'app/components/SVG/SvgLike'
+import SvgComment from 'app/components/SVG/SvgComment'
+import SvgRetwit from 'app/components/SVG/SvgRetwit'
+import { Twit } from 'types/Twit'
+import Image from 'app/components/Image'
+
+export type TwitUnitProps = {
+  data: Twit
+}
+
+const defaultAvatarUrl =
+  'data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBw8PCw0QEQ8PChINDQ0IDQ0NCA8ICggOGBEWFhURExUYHSggGBo0GxMVITEhJSkrLjouFx8zODMtNygtLisBCgoKDQ0NDg0NDisZFRkrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrK//AABEIAGQAZAMBIgACEQEDEQH/xAAbAAEAAwEBAQEAAAAAAAAAAAAAAwQFAgEGB//EADAQAAIBAgMHAwMDBQAAAAAAAAABAgMRBBIxBSEiMkFhcVFSYkKBghMj0XKRkqHB/8QAFQEBAQAAAAAAAAAAAAAAAAAAAAH/xAAUEQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIRAxEAPwD9xAOKlRRi23ZLVgdNlHEbShHdH9x9uX+5QxuNlUulwx9Pd5KgFqptCpLrk7JZSvKbercvLzHIKgn9iWniakdJyXa+aJEANGjtWS5kpd1ws0qGIjNXi0/VdYnzh1Cbi003FrRoD6cFHA45T4ZcMv8AUy8RQAAeNmFtDF/qSsuWL3fLuX9q18tPKtZ8P49TFAAAqABbpYCb1tDs+YCoC9LZz6ST8oq1qEocyt6NcsgIwAB6nZp71biTRu7PxX6kN/NHhl37mCTYOvkqKXTll4A+jB4mCKwdp1M1aXx4EVTqpK8pP1bkclQAAGls/D2Sm9ZcvxRcPIqyXZZT0ihzOCkmnvT1R0AMbEUsk2tbaP3IiL21I74P1TiUSoAADe2dPNRjf6f239gZOHrOMbd7giq7WoJcXDLVmvk7ERUAABtYapmpxfbK/JIY+FxLpt9U9UalLEQlo14fDIipAeOSXVLyypiccldR4n7vpiBBtGpeol7Vl+5UDYKgAAPVG4NTZuHzUr+smAI9sUbSU/VZX5M4+jxNFTg4vro/az56pFxk09zTs0ByATYbDOb9qWsgISWOHm9Ivy1lNSjh4w0X5PmJSKyHg6ntb/KJFODjqnHyjcDXh9mBgg0sRgU7uPC/b9MjOlFptO6tqmVHh7GN2kt93lSPDQ2ThryzvRcvdgauHp5YRj6KwJARQo7Qwf6izR5krf1ovAD5uhQcp5d8bc3xNiEUkktyWiLEqSbvo3q0tSKUWv5A5AAAAACrjsPmjdc0V/kvaWiSFL13dgMbBYR1JdVFc0v+I3oQUUktySskIxSVkrW6I6AAAAAAAAA4dNePBFNWAA56kkKaYAEqikegAAAAAAH/2Q=='
+
+export default function TwitsUnit({
+  data: {
+    id,
+    author,
+    content,
+    likes = 0,
+    comments = 0,
+    retwits = 0,
+    createdAt,
+    updatedAt,
+  },
+}: TwitUnitProps) {
+  const avatar = author.avatar ?? defaultAvatarUrl
+
+  function elapsedDate(time: any) {
+    const date = new Date((time || '').replace(/-/g, '/').replace(/[TZ]/g, ' '))
+    const sec_diff = (new Date().getTime() - date.getTime()) / 1000
+    const day_diff = Math.floor(sec_diff / 86400)
+
+    if (isNaN(day_diff) || day_diff < 0 || day_diff >= 31) return
+
+    return (
+      (day_diff === 0 &&
+        ((sec_diff < 60 && 'just now') ||
+          (sec_diff < 120 && '1 minute ago') ||
+          (sec_diff < 3600 && Math.floor(sec_diff / 60) + ' minutes ago') ||
+          (sec_diff < 7200 && '1 hour ago') ||
+          (sec_diff < 86400 && Math.floor(sec_diff / 3600) + ' hours ago'))) ||
+      (day_diff === 1 && 'Yesterday') ||
+      (day_diff < 7 && day_diff + ' days ago') ||
+      (day_diff === 7 && 'week ago') ||
+      (day_diff < 31 && Math.ceil(day_diff / 7) + ' weeks ago') ||
+      (day_diff === 31 && 'mounth ago') ||
+      (day_diff < 365 && Math.ceil(day_diff / 31) + ' months ago') ||
+      (day_diff === 365 && 'year ago') ||
+      (day_diff > 730 && Math.ceil(day_diff / 365) + ' years ago')
+    )
+  }
```

Comment:
Style of this code do not match anything we wrote. 

- Generally for functions inside components we use lambda (even if it may be moved to utils)
```typescript
const elapsedDate = (time: any): string | undefined => {
```

- We use camelCase
```typescript
const secDiff = ...
const dayDiff = ...
```

- Here optional chaining is used for too big if-else. Maybe try to rewrite to something like that:
```typescript
if (dayDiff === 0) {
  const secDiff = ...
  
  if (secDiff < 60) return 'just now'
  else if (secDiff < 120) return '1 minute ago' //and so on
} else if (dayDiff === 1 {
  return 'Yesterday'
}  //and so on
```
But it might be rewriten to much nicer code

Reply:
Also consider to changing type of `time` to `Date`